### PR TITLE
#1755 | Dependencies update

### PR DESCRIPTION
**Pull request by alexander-vo**
- removed unused dependencies
- dependencies versions updated
#1755

### CODE REVIEW

File: server/server/common/arangodb.py [hunk 1]
```diff
@@ -76,13 +76,13 @@ def __init__(self, app=None):
...
         config = current_app.config['ARANGO_CLIENT']
-        return ArangoClient(host=config['host'], port=config['port'])
+        return ArangoClient(hosts=f'http://{config["host"]}:{config["port"]}')
 
     @property
     def client(self) -> ArangoClient:
         """
         Puts client object in g object.
-        :return: ArangoClient object connected to the databse.
+        :return: ArangoClient object connected to the database.
```
Comment (sujato) [inline server/server/common/arangodb.py:85]: The changed line here throws an error with `make delete-database`. There appears to be a typo in it, but even when that's corrected there's still an error.

**Review by sujato — COMMENTED** [review]
There's an error in arongodb.py.

File: server/server/common/arangodb.py (hunk 1 above)
Comment (alexander-vo) [reply server/server/common/arangodb.py:85]: What kind of error? I just ran it and it's okay...

**Comment by sujato** [comment]
After running `make run-preview-env` it works fine.

**Pull request merged**